man/systemd.exec.xml

```diff
@@ -1653,6 +1653,41 @@ StateDirectory=aaa/bbb ccc</programlisting>
         <xi:include href="version-info.xml" xpointer="v234"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>StateDirectoryQuota=</varname></term>
+        <term><varname>CacheDirectoryQuota=</varname></term>
+        <term><varname>LogsDirectoryQuota=</varname></term>
+
+        <listitem><para>Specifies the storage limits for the directories specified in <varname>StateDirectory=</varname>,
+        <varname>CacheDirectory=</varname>, or <varname>LogsDirectory=</varname> respectively.</para>
+
+        <para>The storage quota is defined in terms of disk blocks and inodes, as per
+        <ulink url="https://man7.org/linux/man-pages/man2/quotactl.2.html">quotactl</ulink>. Takes an absolute size limit in bytes. If the value
+        is suffixed with K, M, G or T, the specified size is parsed as Kilobytes, Megabytes, Gigabytes, or Terabytes (with the base 1024), respectively.
+        If an absolute size limit is specified, only the block quota is set (rounded up to the nearest block). Alternatively, a percentage value may be specified,
+        which applies the same percent quota to both blocks and inodes. Defaults to <constant>off</constant>, in which case no storage limits will be set.</para>
+
+        <para>Only hard limits are set, not soft limits. If the underlying filesystem for the specified directories does not support project quotas,
+        the specified storage limits will not be set. Quotas must be turned on for the limits to be enforced.</para>
```

Comment:
maybe change last sentence to "In addition to enabling quota per unit with these settings it is typically necessary to enable quota on the file system level, as well.".

i.e. as i understand you you are trying to say that there's a 2nd place people need to enable quota on, hence list both places.

(even better would be, to tell people how to actually do that, i.e. the minimal change necessary to get this to work.)